ingestion/src/metadata/ingestion/api/workflow.py

```diff
@@ -227,6 +229,20 @@ def print_status(self) -> int:
             click.echo(self.bulk_sink.get_status().as_string())
             click.echo()
 
+        if self.source.get_status().source_start_time:
+            click.secho(
+                f"Workflow finished in time {pretty_print_time_duration(time.time()-self.source.get_status().source_start_time)} ",
+                fg="bright_cyan",
+                bold=True,
+            )
+            source_sucess = len(self.source.get_status().success)
+            source_failed = len(self.source.get_status().failures)
+            click.secho(
+                f"Success % : {round(source_sucess*100/(source_sucess+source_failed),2)}",
```

Comment:
Can we pls have the `success_percentage` computation as a separated function? This way it will be easier to add that in the `IngestionPipeline` later on 🙏

We could even make this part of the `status` of the workflow as well 🤔

Reply:
Hmm, makes sense